metadata/src/test/java/org/apache/kafka/metadata/storage/FormatterTest.java

```diff
@@ -0,0 +1,502 @@
+//    }
+//
+//    @Test
+//    def testStandaloneModeWithArguments(): Unit = {
+//        val namespace = StorageTool.parseArguments(Array("format", "-c", "config.props", "-t", "XcZZOzUqS4yPOjhMQB6JAY",
+//                "-s"))
+//        val properties: Properties = newSelfManagedProperties()
+//        val logDir1 = "/tmp/other1"
+//        val logDir2 = "/tmp/other2"
+//        properties.setProperty(ServerLogConfigs.LOG_DIRS_CONFIG, logDir1 + "," + logDir2)
+//        properties.setProperty("listeners", "PLAINTEXT://localhost:9092")
+//        val config = new KafkaConfig(properties)
+//        try {
+//            val exitCode = StorageTool.runFormatCommand(namespace, config)
+//            val tempDirs = config.logDirs
+//            tempDirs.foreach(tempDir => {
+//                    val checkpointDir = tempDir + "/" + CLUSTER_METADATA_TOPIC_NAME
+//                    val checkpointFilePath = Snapshots.snapshotPath(Paths.get(checkpointDir), BOOTSTRAP_SNAPSHOT_ID)
+//                    assertTrue(checkpointFilePath.toFile.exists)
+//                    assertTrue(Utils.readFileAsString(checkpointFilePath.toFile.getPath).contains("localhost"))
+//                    Utils.delete(new File(tempDir))
+//            })
+//            assertEquals(0, exitCode)
+//        }
+//        finally{
+//            Utils.delete(new File(logDir1))
+//            Utils.delete(new File(logDir2))
+//        }
+//    }
+//
+//    @Test
+//    def testControllerQuorumVotersWithArguments(): Unit = {
+//        val namespace = StorageTool.parseArguments(Array("format", "-c", "config.props", "-t", "XcZZOzUqS2yQQjhMQB7JAT",
+//                "--controller-quorum-voters", "2@localhost:9092"))
+//        val properties: Properties = newSelfManagedProperties()
+//        val logDir1 = "/tmp/other1"
+//        val logDir2 = "/tmp/other2"
+//        properties.setProperty(ServerLogConfigs.LOG_DIRS_CONFIG, logDir1 + "," + logDir2)
+//        properties.setProperty("listeners", "PLAINTEXT://localhost:9092")
+//        val config = new KafkaConfig(properties)
+//        try{
+//            val exitCode = StorageTool.runFormatCommand(namespace, config)
+//            val tempDirs = config.logDirs
+//            tempDirs.foreach(tempDir => {
+//                    val checkpointDir = tempDir + "/" + CLUSTER_METADATA_TOPIC_NAME
+//                    val checkpointFilePath = Snapshots.snapshotPath(Paths.get(checkpointDir), BOOTSTRAP_SNAPSHOT_ID)
+//                    assertTrue(checkpointFilePath.toFile.exists)
+//                    assertTrue(Utils.readFileAsString(checkpointFilePath.toFile.getPath).contains("localhost"))
+//                    Utils.delete(new File(tempDir))
+//            })
+//            assertEquals(0, exitCode)
+//        }
+//        finally {
+//            Utils.delete(new File(logDir1))
+//            Utils.delete(new File(logDir2))
+//        }
+//    }
```

Comment:
Are you planning to uncomment these tests? If not, let's remove them.

Reply:
will remove